zooniverse/front-end-monorepo
# app-root: Dockerize app-root

## Package
- app-root
- lib-user

## Describe your changes
- adds app-root docker-compose.yml
- updates app-root README with Docker information
- add app-root `/public/robots.txt`
- in app-root package.json and all related instances, rename app-root to fe-root
- update FEM Dockerfile for app-root and lib-user
- update FEM README for app-root Docker info, linting
- update FEM docker-compose.yml with app-root, rearrange apps in alphabetical order

## How to Review
_Helpful explanations that will make your reviewer happy:_
- What Zooniverse project should my reviewer use to review UX? n/a
- What user actions should my reviewer step through to review this PR?
  - from `/front-end-monorepo/`
    - run `docker-compose build`
    - run `docker-compose up`
    - confirm the following load (note authentication does not work in Docker):
      - app-content-pages: http://localhost:3001/about/about
      - app-project: http://localhost:3000
      - app-root: http://localhost:3002/users
    - run `docker-compose down`
  - from `/front-end-monorepo/packages/app-root`
    - run `docker-compose up -d --build`
    - confirm app-root loads at http://localhost:3000/users
    - run `docker-compose down`
- Which storybook stories should be reviewed? n/a
- Are there plans for follow up PR’s to further fix this bug or develop this feature? yes, update GitHub Actions workflows and Kubernetes settings

# Checklist
_PR Creator - Please cater the checklist to fit the review needed for your code changes._
_PR Reviewer - Use the checklist during your review. Each point should be checkmarked or discussed before PR approval._

## General

## Refactoring
